lib/app.js

```diff
@@ -1,14 +1,23 @@
+const cookieParser = require('cookie-parser');
 const express = require('express');
+const cors = require('cors');
 
 const app = express();
 
-// Built in middleware
 app.use(express.json());
+app.use(cookieParser());
+app.use(
+  cors({
+    origin: [
+      'https://glowing-paletas-75ae6b.netlify.app',
+    ],
+    credentials: true,
+  })
+);
 
-// App routes
+app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/tasks', require('./controllers/tasks'));
```

Comment:
since all the routes in this controller use the `authenticate` middleware you can add it here
`app.use('/api/v1/tasks', authenticate, require('./controllers/tasks'));`